feat(changelog): object-storage-added-sse-c-encryption-is-now-in-ga-for- 2024-11-06 (#3952)
scaleway/docs-content: changelog/november2024/2024-11-06-object-storage-added-sse-c-encryption-is-now-in-ga-for-.mdx

---

> title: SSE-C Encryption is now in GA for all Scaleway Regions

Reviewer: ```suggestion
title: SSE-C Encryption is now available in all Scaleway Regions
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-06
category: storage
product: object-storage
---

Server-Side Encryption with Customer-Provided Keys (SSE-C) is now available across all Scaleway regions for new and existing buckets. SSE-C enhances data protection for sensitive or private data stored in Object Storage, allowing to easily encrypt data during upload for added security, while giving you full control over your encryption keys.

> Read our dedicated documentation to get started with SSE-C on Object Storage: https://www.scaleway.com/en/docs/storage/object/api-cli/enable-sse-c/

Reviewer 2: ```suggestion
For more information [read our dedicated documentation to get started with SSE-C on Object Storage](https://www.scaleway.com/en/docs/storage/object/api-cli/enable-sse-c/).
```

> For more information [read our dedicated documentation to get started with SSE-C on Object Storage](https://www.scaleway.com/en/docs/storage/object/api-cli/enable-sse-c/).

Reviewer 3: ```suggestion
For more information [read our dedicated documentation to get started with SSE-C on Object Storage](/storage/object/api-cli/enable-sse-c/).
```
